.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ references:
 
   node_container: &node_container
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:12
```

Comment:
Such change technically requires a major version bump, which is a problem for backporting this to v1. I don't remember when `await` was added to Node. Was it 8 or 10?

Reply:
it was 8, I reverted [v1](https://github.com/AntonBazhal/dynamo2es-lambda/pull/73) back to 8, and I'll revert this one back too.